containers-orchestration/managed-kubernetes/use-public-cloud-load-balancer/private_lb/lb_private_with_openstack_port.yaml

```diff
@@ -0,0 +1,23 @@
+# Only available with a MKS cluster attached to a Private network
+apiVersion: v1
+kind: Service
+metadata:
+  name: octavia-private-with-openstack-port
+  annotations:
+    loadbalancer.ovhcloud.com/class: "octavia"
+    service.beta.kubernetes.io/openstack-internal-load-balancer: "true"
+    # Create a OpenStack port and use this as LB Ip address
+    # openstack port create --network bdd4fc7c-6a27-43d6-940a-8ed51e1fd22c --fixed-ip subnet=187dc936-c4ea-49ca-aff5-0a5eb7062ed9 my-lb-app-port
```

Comment:
These information need to be addded in a README, not a yaml file please

Reply:
fixed